src/server/status_server/mod.rs

```diff
@@ -614,13 +623,27 @@ where
                                 Self::get_config(req, &cfg_controller).await
                             }
                             (Method::POST, "/config") => {
-                                Self::update_config(cfg_controller.clone(), req).await
+                                if check_cert(security_config, x509) {
+                                    Self::update_config(cfg_controller.clone(), req).await
```

Comment:
leave comment to explain what are the endpoints that should call check_cert?

Reply:
Done PTAL